frappe/desk/doctype/auto_repeat/auto_repeat.py

```diff
@@ -97,6 +98,21 @@ def update_status(self, status=None):
 		if status and status != 'Resumed':
 			self.status = status
 
+	def get_auto_repeat_schedule(self):
+		schedule_details = []
+		start_date_copy = self.start_date
```

Comment:
since this schedule will be built dynamically, the start date should be today's date or `start_date`, which ever is later.